client/src/components/SigninButton.svelte

```diff
@@ -1,19 +1,151 @@
 <script lang="ts">
   import Modal from "./Modal.svelte";
   import { authStore as user } from "$stores/auth";
+  import { loginWithGoogle, loginWithMicrosoft, loginWithEmail, onSignOut, createUser } from "$lib/firebase/auth";
 
   // check if the user is logged in with getAuth
   let openSignInModal = false;
+  let email = "";
+  let password = "";
+  let confirmPass = "";
+  let clickedButton = false;
+  let clickedButton2 = false;
+  let errorMessage = "";
+  async function createAccount() {
+    if (password == "") {
+      return;
+    }
+    if (password != confirmPass) {
+      errorMessage = "Password does not match";
+      password = "";
+      confirmPass = "";
+      return;
+    }
```

Comment:
a message should be displayed to the user to explain why nothing happened when they clicked the button

Reply:
on line 19 i set errorMessage to "Password do not match" and I display that to the user, ill move that to the return statment
resolved